src/core/grid-integration.ts

```diff
@@ -19,6 +19,20 @@ import {
 } from './types.js'
 import type { GridApi } from 'ag-grid-community'
 
+// Map AG Grid number operations to our internal number operations
+// Since AG Grid uses generic names, we need to be explicit about number operations
+const AG_GRID_TO_NUMBER_OPERATION_MAP: Record<string, FilterOperation> = {
+  lessThan: 'lessThan',
+  lessThanOrEqual: 'lessThanOrEqual',
+  greaterThan: 'greaterThan',
+  greaterThanOrEqual: 'greaterThanOrEqual',
+  inRange: 'inRange',
+  equals: 'eq',
+  notEqual: 'notEqual',
+  blank: 'blank',
+  notBlank: 'notBlank'
+} satisfies Record<string, FilterOperation>
```

Comment:
You don't need the type, you can just use const and satisfies. I don't remember the order now, but double check with the editor
```suggestion
const AG_GRID_TO_NUMBER_OPERATION_MAP = {
  lessThan: 'lessThan',
  lessThanOrEqual: 'lessThanOrEqual',
  greaterThan: 'greaterThan',
  greaterThanOrEqual: 'greaterThanOrEqual',
  inRange: 'inRange',
  equals: 'eq',
  notEqual: 'notEqual',
  blank: 'blank',
  notBlank: 'notBlank'
} satisfies Record<string, FilterOperation> as const;
```